src/trading/MakeOrder.sol

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2015 Forecast Foundation OU, full GPL notice in LICENSE
+
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controlled.sol';
+import 'ROOT/libraries/trading/PlaceOrder.sol';
+import 'ROOT/libraries/ReentrancyGuard.sol';
+import 'ROOT/trading/IMakeOrder.sol';
+import 'ROOT/trading/Trading.sol';
+
+
+contract MakeOrder is Controlled, ReentrancyGuard {
+    using PlaceOrder for PlaceOrder.Data;
+
+    function makeOrder(Trading.TradeTypes _type, uint256 _attoshares, int256 _displayPrice, IMarket _market, uint8 _outcome, bytes32 _betterOrderId, bytes32 _worseOrderId, uint256 _tradeGroupId) onlyInGoodTimes nonReentrant external returns (bytes32 _orderId) {
+        PlaceOrder.Data memory _placeOrderData = PlaceOrder.create(controller, msg.sender, _outcome, _type, _attoshares, _displayPrice, _market, _betterOrderId, _worseOrderId, _tradeGroupId);
+
+        uint256 _fxpMoneyEscrowed;
+        uint256 _fxpSharesEscrowed;
+        if (_placeOrderData.tradeType == Trading.TradeTypes.Ask) {
```

Comment:
Building on the comment above about `placeBid` and `placeAsk` functions, consider having a function on `Order` that is `escrowFunds` that contains this logic and then calls the appropriate method (`escrowFundsForBid` or `escrowFundsForAsk`) internally.  Then the caller (this code) would merely need to call `escrowFunds` and the `Order` class would do the rest.